### ISSUE TITLE
State of sim missing an "of" 

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#122]
State of sim pdom paragraph says: "Currently, beaker has ___ moles drink mix..." 

and should say: 
"Currently, beaker has ___ moles of drink mix ..."

**Comment by terracoda** [comment]
@twant, this might be covered in #123. 
You, of course, can arrange the phrases and their parameters as you see best programmatically. I put {{curly brackets}} around the things that I felt were sub-strings containing parameters. The brackets are just a guide at this point.

I hope all is clear in #123 :-)

**Comment by terracoda** [comment]
@twant, I think I cleaned up the brackets in to something more logical in https://github.com/phetsims/molarity/issues/123#issuecomment-509726059, hopefully this helps.

**Comment by twant** [comment]
I think you're right that this is covered, @terracoda. Closing this one!
